doc/cabal-package-description-file.rst

```diff
@@ -2693,87 +2693,137 @@ Starting with Cabal-2.2 it's possible to use common build info stanzas.
 *Source code* repository marker
 -------------------------------
 
-.. pkg-section:: source-repository
+.. pkg-section:: source-repository notation-variant
     :since: 1.6
 
-A marker that points to the *source code* for this package within a
-**source code repository**.
+A metadata marker that helps users of your package find its source code.
+This marker points to a *source code* snapshot within a
+version control systems (VCS) **repository** like Git.
 
-There are two kinds. You can specify one or the other or both at once:
+Here are some examples that point to snapshots of the Cabal library in
```

Comment:
I think moving the examples before the definition of the kinds (this vs head) is a bad idea: without knowing about this and head examples make little sense.